admin_manual/installation/linux_installation.rst

```diff
@@ -2,46 +2,59 @@
 Preferred Linux Installation Method
 ===================================
 
+We recommend the installation from the tar ball for large productive environments 
```

Comment:
It would be much better to write
"For production environment, we recommend ownCloud installation from tarball"